base/version_git.sh

```diff
@@ -82,6 +84,16 @@ if [ -z "$fork_master_timestamp" ]; then
     fork_master_timestamp="0"
 fi
 
+build_system_directory="../.buildkite"
+if [[ -d "${build_system_directory:?}" ]]; then
```

Comment:
```suggestion
if [[ -d "${build_system_directory:?}/.git" ]]; then
```

@staticfloat Perhaps this is a more relevant check?

Reply:
Yeah.  I also think that since `build_system_directory` is defined just above this line, we don't really need the `:?` suffixes.  Not that they hurt, just, they look like the file was written with `build_system_directory` intended to be user-provided.

Reply:
I've removed the `:?` suffixes.